### PR TITLE
[OpenVINO-EP] Disabled div unit tests for GPU

### PR DESCRIPTION
**Pull request by suryasidd**
**Description**: Disabled div unit tests on GPU_FP32 and GPU_FP16

**Motivation and Context**
-Disabling Div unit tests on OpenVINO-EP for now because these are not supported on the GPU plugin. These will be supported in the next release.

### CODE REVIEW

File: onnxruntime/test/python/onnx_backend_test_series.py [hunk 1]
```diff
@@ -160,6 +160,9 @@ def create_backend_test(testname=None):
         if c2.supports_device('NGRAPH'):
             current_failing_tests = current_failing_tests + ('|^test_clip*',)
 
+        if c2.supports_device('OPENVINO_GPU_FP32'):
```
Comment (HectorSVC) [inline onnxruntime/test/python/onnx_backend_test_series.py]: >OPENVINO_GPU_FP32 [](start = 31, length = 17)

shouldn't we add "OPENVINO_CONFIG_GPU_FP16" as well?

Comment (jywu-msft) [reply onnxruntime/test/python/onnx_backend_test_series.py]: yes, add OPENVINO_GPU_FP16 as well.

**Comment by HectorSVC** [comment]
OpenVINO_GPUDevice_Nightly build passed with this PR https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=69489

File: onnxruntime/test/python/onnx_backend_test_series.py (hunk 1 above)
Comment (suryasidd) [reply onnxruntime/test/python/onnx_backend_test_series.py]: Yes i missed this. I will add this change

**Pull request merged**